pulsar-broker/src/main/java/org/apache/pulsar/broker/namespace/NamespaceService.java

```diff
@@ -1342,7 +1342,11 @@ public void addNamespaceBundleOwnershipListener(NamespaceBundleOwnershipListener
             }
         }
         pulsar.runWhenReadyForIncomingRequests(() -> {
-            getOwnedServiceUnits().forEach(bundle -> notifyNamespaceBundleOwnershipListener(bundle, listeners));
+            try {
+                getOwnedServiceUnits().forEach(bundle -> notifyNamespaceBundleOwnershipListener(bundle, listeners));
```

Comment:
I think we should process the possible exception from `notifyNamespaceBundleOwnershipListener` as well. Currently, if one listener fails, the following listeners will be skipped. We'd better catch the `Throwable` for `notifyNamespaceBundleOwnershipListener` as well.

Reply:
The `notifyNamespaceBundleOwnershipListener ` method already caught exceptions. See:
 https://github.com/apache/pulsar/blob/76f16e811beb4f48fb2ae5c46558b74d333c7d60/pulsar-broker/src/main/java/org/apache/pulsar/broker/namespace/NamespaceService.java#L1362-L1368